gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -1,5 +1,22 @@
 """Script to create release sites HT for v4 genomes."""
+
```

Comment:
I think a commit slipped in here. Or am I missing where the above output is used? Since this is a PR to merge CADD, I think any release changes are out of scope.

Reply:
Sorry, that was my on-going script. I removed it from this PR.